src/Network/Minio/Sign/V4.hs

```diff
@@ -98,7 +98,9 @@ signV4AtTime ts ci ri =
     outHeaders = authHeader : headersWithDate
     timeBS = awsTimeFormatBS ts
     dateHeader = (mk "X-Amz-Date", timeBS)
-    hostHeader = (mk "host", encodeUtf8 $ connectHost ci)
+    hostHeader = (mk "host", encodeUtf8 $
+                   format "{}:{}" $
+                   [connectHost ci, show $ connectPort ci])
```

Comment:
Break lines only when the are too wide. Here it looks like the the line with `format` can be joined with the previous line.